code/modules/events/disease_outbreak.dm

```diff
@@ -98,66 +172,153 @@
 	disease_event.disease_candidates.Cut() //Clean the list after use
 
 	if(!max_symptoms)
-		max_symptoms = 3 + max(FLOOR((world.time - control.earliest_start)/6000, 1),0) //3 symptoms at 20 minutes, plus 1 per 10 minutes.
-		max_symptoms = clamp(max_symptoms, 3, 8) //Capping the virus symptoms prevents the event from becoming "smite one poor player with an -12 transmission hell virus" after a certain round length.
+		max_symptoms = rand(ADV_MIN_SYMPTOMS, ADV_MAX_SYMPTOMS) // Pick how many symptoms
 
 	if(!max_severity)
-		max_severity = 3 + max(FLOOR((world.time - control.earliest_start)/6000, 1),0) //Max severity doesn't need clamping
+		var/rng_severity = rand(1, 100) // Pick the severity
+		if(rng_severity < ADV_RNG_LOW)
+			max_severity = ADV_DISEASE_MEDIUM
 
-	var/datum/disease/advance/advanced_disease = new /datum/disease/advance/random(max_symptoms, max_severity)
+		else if(rng_severity < ADV_RNG_MID)
+			max_severity = ADV_DISEASE_HARMFUL
 
-	var/list/name_symptoms = list() //for feedback
+		else
+			max_severity = ADV_DISEASE_DANGEROUS
+
+	var/datum/disease/advance/advanced_disease = new /datum/disease/advance/random/event(max_symptoms, max_severity)
+
+	var/list/name_symptoms = list() // For feedback
 	for(var/datum/symptom/new_symptom in advanced_disease.symptoms)
 		name_symptoms += new_symptom.name
 
 	var/mob/living/carbon/human/victim = pick_n_take(afflicted)
 	if(victim.ForceContractDisease(advanced_disease, FALSE))
-		message_admins("An event has triggered a random advanced virus outbreak on [ADMIN_LOOKUPFLW(victim)]! It has these symptoms: [english_list(name_symptoms)]")
-		log_game("An event has triggered a random advanced virus outbreak on [key_name(victim)]! It has these symptoms: [english_list(name_symptoms)].")
+		message_admins("An event has triggered a random advanced virus outbreak on [ADMIN_LOOKUPFLW(victim)]! It has these symptoms: [english_list(name_symptoms)]. Transmissibility is [advanced_disease.spread_text].")
+		log_game("An event has triggered a random advanced virus outbreak on [key_name(victim)]! It has these symptoms: [english_list(name_symptoms)]. Transmissibility is [advanced_disease.spread_text].")
```

Comment:
```suggestion
		message_admins("An event has triggered a random advanced virus outbreak on [ADMIN_LOOKUPFLW(victim)]! It has these symptoms: [english_list(name_symptoms)]. It is transmissable by [advanced_disease.spread_text].")
		log_game("An event has triggered a random advanced virus outbreak on [key_name(victim)]! It has these symptoms: [english_list(name_symptoms)]. It is transmissable by [advanced_disease.spread_text].")
```

couple these with the changes to spread_text i include somewhere else in here and it'll look chef's kiss.

Reply:
adjusted it a bit in the spread_text, sure